src/main/java/Duke.java

```diff
@@ -1,10 +1,70 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+        /*
+         * Duke greetings
+         */
+        String logo = " ____        _        \n" + "|  _ \\ _   _| | _____ \n" + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n" + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        System.out.println("    Hello! I'm Duke\n    What can I do for you?");
+        System.out.println("    ____________________________________________________________\n");
+
+        /*
+         * Takes in user inputs, and echoes them. If user input == "bye", then program
+         * exits
+         */
+        Scanner userInput = new Scanner(System.in);
+        String currentInput = userInput.nextLine();
+        // while (!"bye".equals(currentInput)) {
+        // System.out.println("
+        // ____________________________________________________________");
+        // System.out.println(" " + currentInput);
+        // System.out.println("
+        // ____________________________________________________________\n");
+        // currentInput = userInput.nextLine();
+        // }
+
+        TaskManager t = new TaskManager();
+        while (!"bye".equals(currentInput)) {
+            System.out.println("    ____________________________________________________________");
+
+            switch (currentInput) {
+            case "list":
+                t.listTasks();
+                break;
+            default:
+                System.out.println("    " + "added: " + currentInput);
+                t.addTask(currentInput);
+                break;
+            }
+            System.out.println("    ____________________________________________________________\n");
+            currentInput = userInput.nextLine();
+
+            if (currentInput.startsWith("mark")) {
+                String[] operation = currentInput.split(" ");
+                t.markTask(operation[1]);
+            } else if (currentInput.startsWith("unmark")) {
```

Comment:
It would be good to move the else if statement to the next line for better readability